### PR TITLE
feat: skip for now button

### PR DESCRIPTION
**Pull request by HumanoidSandvichDispenser**
Duplicate of #1 since the fork was in a detached state. This happened when this repository was made public, and [the fork turned into its own standalone repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/what-happens-to-forks-when-a-repository-is-deleted-or-changes-visibility#changing-a-private-repository-to-a-public-repository).

This PR adds a skip for now button, which is currently a work-in-progress.

It skips the question without deleting it from the question pool, much like clicking the Answer tab while in the Answer tab.

There is currently nothing that pops up confirmation dialog if the user has that enabled.

### CODE REVIEW

File: .dir-locals.el [hunk 1]
```diff
@@ -0,0 +1,4 @@
+((javascript-mode . ((js-indent-level . 2)
+                     (evil-shift-width . 2)
+                     (fill-column . 100) ;; NOTE: 80 or 100?
+                     (lsp . 1))))
```
Comment (aprzn123) [inline .dir-locals.el:4]: Probably makes sense to add this file to the .gitignore

Comment (HumanoidSandvichDispenser) [reply .dir-locals.el:4]: I initially included this file to help enforce code styles consistent with the rest of the repo for any contributor who is using Emacs. Do you have more suggestions on how to best handle this?

Comment (aprzn123) [reply .dir-locals.el:4]: I feel like emacs configs are varied enough that it doesn't make sense to have a universal one like this. I see what you're saying there, though

Comment (HumanoidSandvichDispenser) [reply .dir-locals.el]: I see. I untracked the file now.

**Comment by HumanoidSandvichDispenser** [comment]
Here's what the skip for now button looks like:

image

image

**Comment by HumanoidSandvichDispenser** [comment]
That looks like a lot of commits. I'll squash them if you need me to.

**Comment by aprzn123** [comment]
Any idea why the Skip for Now button looks slightly different from the others in the Chromium screenshot? It looks fine in the Firefox one, but not quite sure what's going on there. Other than that, lgtm.

**Comment by HumanoidSandvichDispenser** [comment]
Is it the background color that's different? If the button is focused, then the background will darken. Normally it should look like this 
image

**Comment by aprzn123** [comment]
Oh, it was focused. That makes sense. Thanks for working on this!

**Pull request merged**